test/apps/angular-v14/package.json

```diff
@@ -40,6 +40,9 @@
     "lite-server": "^2.6.1",
     "typescript": "~4.8.3"
   },
+  "resolutions": {
+    "core-js": "3.40.0"
+  },
```

Comment:
Based on the `yarn.lock` file below I don't think this one is needed since the semver resolve to this once once you let it upgrade to the latest